e2e/e2e/connection.py

```diff
@@ -250,3 +254,89 @@ def query_connection_end(c: Config, chain_id: ChainId, conn_id: ConnectionId) ->
     l.debug(f'Status of connection end {conn_id}: {res}')
 
     return res
+
+# =============================================================================
+# Passive CONNECTION relayer tests
+# =============================================================================
+
+def verify_state(c: Config,
+    ibc1: ChainId, ibc0: ChainId,
+    ibc1_conn_id: ConnectionId):
+
+    strategy = toml.load(c.config_file)['global']['strategy']
+    # verify connection state on both chains, should be 'Open' for 'all' strategy, 'Init' otherwise
+
+    if strategy == 'all':
```

Comment:
Just to clarify: the intended use-case for this is for development-time, when running the e2e tests locally. The strategy is `packets` in the ci config, so these new tests are not expected to run in the CI, right?

Reply:
The e2e test suite does work with `strategy = 'all'`, I've changed it in CI to reflect that.